src/app/app.module.ts

```diff
@@ -33,7 +40,8 @@ import { LoaderInterceptor } from './interceptors/loader.interceptor';
     ],
```

Comment:
`FormsModule` is imported at line 2 but not mentioned in the `@NgModule` imports. Please add `FormsModule` to be able to compile the application.

Reply:
adapted